### PR TITLE
update to be at the same level as en.js

### PR DESCRIPTION
**Pull request by devzwf**
this is to align with with the lastest change in en.js

### CODE REVIEW

File: scripts/i18n/fr.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-    'error_fetching_image_results': 'Erreur lors de la récupération des résultats d\'images :',
-    'subscription_deleted': 'Abonnement supprimé',
-    'error_deleting_subscription': "Erreur lors de la suppression de l'abonnement",
-    'failed_to_load_subscription': "Impossible de charger l'abonnement",
-    'edit_subscription': "Modifier l'abonnement",
-    'add_subscription': "Ajouter un abonnement",
+    error_reloading_subscription: "Erreur lors du rechargement de l'abonnement :",
+    error_fetching_image_results: "Erreur lors de la récupération des résultats d'images :",
+    subscription_deleted: "Abonnement supprimé",
+    error_deleting_subscription: "Erreur lors de la suppression de labonnement",
```
Comment (ellite) [inline scripts/i18n/fr.js:6]: Should be l'abonnement

File: scripts/i18n/fr.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+    failed_to_load_subscription: "Impossible de charger labonnement",
```
Comment (ellite) [inline scripts/i18n/fr.js:7]: l'abonnement

File: scripts/i18n/fr.js [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
+    edit_subscription: "Modifier labonnement",
```
Comment (ellite) [inline scripts/i18n/fr.js:8]: l'abonnement

File: scripts/i18n/fr.js [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
-    'save_currency': 'Enregistrer la devise',
-    'delete_currency': 'Supprimer la devise',
-    'failed_remove_currency': 'Échec de la suppression de la devise',
-    'failed_save_currency': 'Échec de l\'enregistrement de la devise',
-    'cant_disable_payment_in_use': 'Impossible de désactiver le paiement en cours d\'utilisation',
-    'failed_save_payment_method': 'Échec de l\'enregistrement de la méthode de paiement',
-    'unknown_error': 'Erreur inconnue, veuillez réessayer.',
-    'error_saving_notification_data': 'Erreur lors de l\'enregistrement des données de notification',
-    'error_sending_notification': 'Erreur lors de l\'envoi de la notification',
+    network_response_error: "La réponse du réseau nétait pas correcte",
```
Comment (ellite) [inline scripts/i18n/fr.js:11]: n'était

**Review by ellite — CHANGES_REQUESTED** [review]
I believe there were some ' removed by mistake.

**Comment by devzwf** [comment]
let me review and get back

**Review by devzwf — COMMENTED** [review]
should be good now

**Review by ellite — APPROVED** [review]

**Pull request merged**